mathics/core/builtin.py

```diff
@@ -244,6 +244,7 @@ def contribute(self, definitions: Definitions, is_pymodule=False):
 
         option_syntax = "Warn"
 
+        assert self.options is not None
```

Comment:
then `options` is not `Optional`, right?

Reply:
True, I had just copied this from BaseElement